src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/UpToDateCheckItemComparer.cs

```diff
@@ -2,16 +2,25 @@
 
 namespace Microsoft.VisualStudio.ProjectSystem.UpToDate
 {
-    internal class UpToDateCheckItemComparer : IEqualityComparer<(string, string, CopyToOutputDirectoryType)>
+    internal class UpToDateCheckItemComparer : IEqualityComparer<(string Path, string Link, CopyToOutputDirectoryType CopyType)>
```

Comment:
Nit, I think Rolsyn/Project-System as a whole decided that naming is lower case for tuple element names. You've mixed casing above and here.

Reply:
Fixed. I prefer lowercase too. Somehow I thought I was preserving existing convention, but now I can't find it elsewhere.